app/kafka/manager/ClusterManagerActor.scala

```diff
@@ -294,6 +295,27 @@ class ClusterManagerActor(cmConfig: ClusterManagerActorConfig)
         }
         result.map(CMCommandResults.apply) pipeTo sender()
 
+      case CMManualPartitionAssignments(assignments) =>
+        implicit val ec = longRunningExecutionContext
+        val result = Future {
+          modify {
```

Comment:
This looks to be duplicating 279-295, can we move to a method so we can reuse in both places.